packages/core/src/transport/httpRequest.spec.ts

```diff
@@ -63,3 +63,35 @@ describe('httpRequest', () => {
     expect(server.requests.length).toEqual(1)
   })
 })
+
+describe('httpRequest parameters', () => {
+  const BATCH_BYTES_LIMIT = 100
+  let server: sinon.SinonFakeServer
+  let sendBeaconSpy: jasmine.Spy<(url: string, data?: BodyInit | null) => boolean>
+  beforeEach(() => {
+    server = sinon.fakeServer.create()
+    sendBeaconSpy = jasmine.createSpy()
+    navigator.sendBeacon = sendBeaconSpy
+  })
+
+  afterEach(() => {
+    server.restore()
+  })
+
+  it('should add batch_time', () => {
+    const request = new HttpRequest('https://my.website', BATCH_BYTES_LIMIT, true)
+
```

Comment:
what about having a test ensuring that we don't have batch time when `withBatchTime=false`?